src/main/java/uk/gov/hmcts/darts/audio/entity/MediaRequestEntity.java

```diff
@@ -57,6 +58,10 @@ public class MediaRequestEntity extends CreatedModifiedBaseEntity {
     @JoinColumn(name = HEARING_ID_COLUMN_NAME, nullable = false)
     private HearingEntity hearing;
 
+    @ManyToOne(optional = false)
```

Comment:
I'm not entirely sure why the other @ManyToOne relationships here don't have LAZY fetch - but thought we could do that here
`@ManyToOne(fetch = FetchType.LAZY, optional = false)`

Reply:
Agree, updated.